cypress/fixtures/community_links.json

```diff
@@ -0,0 +1,44 @@
+{
+  "links": [
```

Comment:
Put this fixture within the test itself, easier to read one file than reading 2 files. No need to have it standalone as this fixture is not shared either. This is the biggest anti-pattern in programming today.

For context, this was discussed in various DeFiCh projects, mostly recently: https://github.com/DeFiCh/salmon/pull/35#discussion_r649011962

Reply:
ok copy, updated